funds/privacy_guides/privacy_guides_funds.md

```diff
@@ -0,0 +1,163 @@
+
+* The Fund has no remaining assets or a negative balance. MAGIC Grants may close the Fund with 30 days’ notice if not rectified in that time.
+* If applicable, the Fund has fewer assets than the minimum annual fee 30-60 days before the next minimum fee is due. MAGIC Grants may announce at least 30 days before the fee is due its intention to close the Fund, giving the committee a chance to determine how to disburse the remaining assets before the fee is charged.
+* MAGIC Grants issues a notice at least 1 year in advance that it wishes to close the Fund, giving the committee a chance to determine how to disburse the remaining assets.
+* The committee votes unanimously to close the Fund. MAGIC Grants may optionally also require a Voter majority vote.
+* The committee votes by majority vote to “split” the Fund into several Funds. This is intended for major Project changes or splits (eg: Bitcoin/Bitcoin Cash). MAGIC Grants may optionally “freeze” the existing Fund while it is determining how to allocate assets among the two new Funds. MAGIC Grants may deny the formation of new Funds, preventing the “split” in effect. Should MAGIC Grants agree to “split” the Fund, Voters must agree by majority vote to 1) close the existing Fund, 2) open two (or more, as applicable) new Funds under new charter documents, and 3) vote to approve the allocation of assets among the new Funds. The new Funds will then undergo the typical Fund startup processes of selecting Voters and electing a committee, subject requirements listed in those new Fund charter documents.
+
+## Renaming the Fund
+
+The committee can vote by majority vote to rename the Fund, which must be approved by MAGIC Grants.
+
+## Changes to this Document
+
+Unless other restrictions apply to the relevant sections, the committee may modify this document by majority vote. MAGIC Grants must approve the changes, and it may require a Voter majority vote to approve the changes even if this is not explicitly required by the existing language. Unless waived by MAGIC Grants, proposed changes should be announced publicly at least 30 days in advance of the change.
+
+Should any portion of this document affect the legal standing of MAGIC Grants, MAGIC Grants may change the relevant portions without any other approvals and without notice.
+
+## MAGIC Grants Existing Board
+
+[Link](/about/)
+
+## Contact Information
+
+MAGIC Grants: [info@magicgrants.org](mailto:info@magicgrants.org)
+
+MAGIC **Project** Fund (emails all committee members): [<>@magicgrants.org](mailto:<>@magicgrants.org)
+
+## Appendix A - Restrictions on Grant Disbursements
+
+In addition to all the normal restrictions on grant disbursements, this MAGIC Fund shall have the following additional restrictions on grant disbursements:
+
+* None
+
+## Appendix B - MAGIC Fund Voter Modifications
+
+Modifications to MAGIC Fund Voters:
+
+* There are no MAGIC Fund Voters for this MAGIC Fund.
```

Comment:
This is how we want it, but some sections refer to "Voter majority votes" (L120, L154 for example), so how would these work? Is it explicitly said somewhere that "Committee Members" are also "Voters," or can we explicitly say that if not?

Reply:
I will take out other references to the Voters, since that does not apply